Mixpanel/MPUITableViewBinding.m

```diff
@@ -122,4 +122,18 @@ - (UITableView *)parentTableView:(UIView *)cell {
     return nil; // this view is not within a tableView
 }
 
+- (BOOL)isEqual:(id)other {
```

Comment:
Do you think we need this? It seems to me that it does the same as its super class `isEqual`.